src/platform/plugins/shared/dashboard/public/dashboard_container/component/viewport/_print_viewport.scss

```diff
@@ -25,6 +25,15 @@ is being formed. This can result in parts of the vis being cut out.
   }
 }
 
+.dshDashboardViewport--print {
+  .kbnGridRow {
+    display: block !important;
+  }
+  .kbnGridPanel {
+    height: 100% !important;
+  }
+}
```

Comment:
It might be controversial that I am targeting `kbn-grid-layout` class names here - but considering that this is **only** used for reporting, and I want to keep the layout engine agnostic of reporting, I've opted to do this instead of adding the style to the `kbn-grid-layout` package. It works the same way as it used to, so hopefully this is okay 🙈